CRM/Civicase/Hook/PostProcess/CaseCategoryPostProcessor.php

```diff
@@ -43,6 +43,13 @@ public function run($formName, CRM_Core_Form &$form) {
     elseif ($formAction == CRM_Core_Action::DELETE) {
       $handler->onDelete($categoryName);
     }
+
+    // Flush all caches using the API.
+    // Skip triggers and sessions to minimize impact.
+    civicrm_api3('System', 'flush', [
+      'triggers' => 0,
+      'session' => 0,
+    ]);
```

Comment:
@ivan-compucorp , do we need the extra parameters? Won't `civicrm_api3('System', 'flush')` be just enough?